sources/fips.json

```diff
@@ -3,7 +3,10 @@
 	"fullname": "Federal Information Processing Standards",
 	"name": "fips",
 	"prefix": "fips",
-    	"key": "code",
+	"key": "code",
 	"url": "http://www.nist.gov/itl/fips.cfm",
-	"license": "https://www.usa.gov/government-works"
-}
+	"license": "https://www.usa.gov/government-works",
+	"license_type": "CC0",
+	"license_text": "Most U.S. government creative works such as writing or images are copyright-free.",
```

Comment:
This license text is taken from the existing license URL: https://www.usa.gov/government-works. It is the most straight-forward text describing the license restrictions for this US govt source, but I can add additional/different wording if there is more appropriate language on the website.. does another sentence/paragraph better describe the license?